azure-quantum/azure/quantum/target/microsoft/qio/simulated_annealing.py

```diff
@@ -16,10 +16,10 @@
 
 
 class SimulatedAnnealing(Solver):
-    target_names = [
+    target_names = (
```

Comment:
Is this change intended? Just asking because the code is out of the scope for the PR tilte.

Reply:
It is not related. I was fixing a bug where a target_name was just a single element and not being treated as a list, and saw that this was the only target_names that was a list instead of a tuple. I see that the bug was independently fixed while I was working on this in the same way.